contracts/src/v0.8/automation/dev/v2_3/AutomationRegistrar2_3.sol

```diff
@@ -356,6 +358,10 @@ contract AutomationRegistrar2_3 is TypeAndVersionInterface, ConfirmedOwner, IERC
     }
     bytes32 hash = keccak256(abi.encode(params));
 
+    if (s_pendingRequests[hash].admin != address(0)) {
+      revert DuplicateEntry();
```

Comment:
can we add a foundry test for the duplicate registration scenario?

Reply:
will do.